flagging_site/templates/index.html

```diff
@@ -1,30 +1,82 @@
 {% extends "base.html" %}
-{% block title %}Water Quality Flagging Overview{% endblock %}
+{% block title %}Flagging Program - Live Water Quality Alerts{% endblock %}
 {% block content %}
-    <h2>Overview</h2>
-    <p>
-        The following predicted concentrations of Fecal Indicator Bacteria (FIB) are
-        based on modeled correlations between FIB and weather or water conditions.
-    </p>
-    <p>
-        Flagging is based on safety thresholds for boating. The weather and water
-        data is sourced from the CESN weather station at Community Boating and the
-        USGS Waltham river flow gauge.
-    </p>
-    <h2>Current Water Quality</h2>
-    <p>
-        Water quality flags, with forecasted E.coli concentrations and
-        probabilities (updated hourly):
-    </p>
-    {% for reach, flag in flags.items() %}
-    <div>
-        Reach {{ reach }}
-        {% if flag %}
-            <img class="flag" src="{{ url_for('static', filename='images/blue_flag.jpg') }}" />
+<h2>Project Overview</h2>
+<!-- Older language
+<p>
+    The following predicted concentrations of Fecal Indicator Bacteria (FIB) are
+    based on modeled correlations between FIB and weather or water conditions.
+</p>
+<p>
+    Flagging is based on safety thresholds for boating. The weather and water
+    data is sourced from the CESN weather station at Community Boating and the
+    USGS Waltham river flow gauge.
+</p> -->
+<!-- Newer language -->
+<p>
+    CRWA's Flagging Program uses a system of color-coded flags to indicate whether or not the river's water quality is
+    safe for boating at eleven boating locations between Watertown and Boston. Flag colors are based on E. coli and
+    cyanobacteria (blue-green algae) levels; blue flags indicate suitable boating conditions and red flags indicate
+    potential health risks.
+</p>
+<p>
+    CRWA estimates E. coli bacteria levels using models that take into account a variety of environmental factors
+    including rainfall and river flow. CRWA collects water samples weekly to verify the model forecasts. Cyanobacteria
+    monitoring is conducted every 1-2 weeks.
+</p>
+<br />
+<!-- Is this needed? CRWA's website links to a 2019 report using a button 
+<p>
+    Subscribe to water quality alerts through the sign up form below.
+</p> -->
+<!-- optional 2019 report button here? -->
```

Comment:
I agree, this probably isn't needed.

As a note, the issue with HTML comments `<!-- comment here -->` is that they render in the HTML. But usually you don't want that. I would instead preference utilizing Jinja comment styles: `{# comment here #}` because these do not render in the HTML. If we were working in PHP instead of Jinja, the same general concept would be true: you'd preference `/* comment here */` inside PHP blocks over using the HTML comments.

Basically, it's preferable to not render comments in production. Except in circumstances like, for example in a lot of modern JS framework such as React and Angular, HTML comments are utilized as nodes within the document's object model and thus have a functional purpose inside of the rendered page. But that's totally different than a real, bona fide comment.